frontend/js/src/user-feed/UserFeed.tsx

```diff
@@ -419,6 +468,59 @@ export default function UserFeedPage() {
           />
         );
       }
+      return (
+        <>
+          {!isSubEvent ? (
+            <ListenControl
+              title="Thanks"
+              text=""
+              icon={faHandshake}
+              buttonClassName="btn btn-link btn-xs"
+              action={() => {
+                NiceModal.show(ThanksModal, {
+                  original_event_id: event.id!,
+                  original_event_type: event.event_type,
+                });
+              }}
+              dataToggle="modal"
+              dataTarget="#ThanksModal"
+            />
+          ) : (
+            null
+          )} 
```

Comment:
I like this conditional rendering format better, with a boolean check followed by the component, rather than having to return `null` or `undefined` with the ternary operator
```suggestion
          {!isSubEvent && (
            <ListenControl
              title="Thanks"
              text=""
              icon={faHandshake}
              buttonClassName="btn btn-link btn-xs"
              action={() => {
                NiceModal.show(ThanksModal, {
                  original_event_id: event.id!,
                  original_event_type: event.event_type,
                });
              }}
              dataToggle="modal"
              dataTarget="#ThanksModal"
            />
          )} 
```